Mathlib/CategoryTheory/IsomorphismClasses.lean

```diff
@@ -42,14 +42,14 @@ def isomorphismClasses : Cat.{v, u} ⥤ Type u where
   map {C D} F := Quot.map F.obj fun X Y ⟨f⟩ => ⟨F.mapIso f⟩
   map_id {C} := by  -- Porting note: this used to be `tidy`
     dsimp; apply funext; intro x
-    apply x.recOn  -- Porting note: `induction x` not working yet
+    apply @Quot.recOn _ _ _ x
```

Comment:
I think the `intro x; apply` sequence should just be `rintro ⟨x⟩`, here and at 52.

(`refine x.recOn ?_ ?_` probably would work too, but `rintro` is cleaner, if it works.)

Reply:
Neither of those work.